bin/cron/build_contact_rollups

```diff
@@ -0,0 +1,428 @@
+#!/usr/bin/env ruby
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require_relative '../../lib/state_abbr'
```

Comment:
Literally every other user of this file required it exactly this way, so I did as they did. But it turns out it doesn't need to be that way. Changed to absolute path.